ios/FluentUI/Notification/MSFNotification.swift

```diff
@@ -196,6 +196,12 @@ import UIKit
     }
 
     @objc public static var allowsMultipleToasts: Bool = false
+
+    /// Only to be used when `isFlexibleWidthToast` is `true`
+    /// A set of constraints to be applied to the `MSFNotification`
+    /// in `show` after it has been added to the view hierarchy.
+    @objc public var flexibleWidthConstraints: [NSLayoutConstraint]?
+
     var isFlexibleWidthToast: Bool
```

Comment:
Should `isFlexibleWidthToast` be public instead of internal so it can be set/seen outside of the init?